README.md

```diff
@@ -16,7 +16,8 @@ later time. The application currently runs only using a Command Line Interface.
 2. Type these commands in the command prompt:
    - gradle build
    - gradle shadowJar
-   - java -jar ./build/libs/mike-all.jar
+   - java -jar ./build/libs/mike-all.jar (launches using the gui)
+   - java -jar ./build/libs/mike-all.jar cli (launches using a command line interface)
```

Comment:
I would have these on the same line. Something like java -jar ./build/libs/mike-all.jar (for GUI) and the java -jar ./build/libs/mike-all.jar cli (for CLI)